src/cljs/rems/application.cljs

```diff
@@ -62,6 +62,16 @@
              (for [resource resources]
                [:li (localized (:catalogue-item/title resource))]))])))
 
+(defn- blacklist-warning [application]
+  (let [blacklist (:application/blacklisted-users application)]
+    (when (not (empty? blacklist))
+      [:div.alert.alert-danger
+       (text :t.form/alert-blacklisted-users)
+       (into [:ul]
+             (for [entry blacklist]
+               [:li (get-member-name (:blacklist/user entry))
+                ": " (get-in entry [:blacklist/resource :resource/ext-id])]))])))
```

Comment:
I think from the UX perspective it doesn't make sense to list here the resource ids but we should instead use the catalogue items that are present in the application and their names.

Reply:
did this in the frontend